plonky2/src/plonk/circuit_builder.rs

```diff
@@ -343,6 +343,13 @@ impl<F: RichField + Extendable<D>, const D: usize> CircuitBuilder<F, D> {
         HashOutTarget::from_vec(self.add_virtual_targets(4))
     }
 
+    /// Registers a new `HashOutTarget` as a public input. `NUM_HASH_OUT_ELTS`
+    /// being hardcoded to 4, it internally adds 4 virtual targets in a
+    /// vector fashion.
```

Comment:
Nit (not likely, but in the eventuality it was to change in the future), perhaps would be easier to just have:

```suggestion
    /// Registers a new `HashOutTarget` as a public input, adding
    /// internally `NUM_HASH_OUT_ELTS` virtual targets.
```

Reply:
(also, they are _not_ added in vector fashion, see comment below)